L1Trigger/Phase2L1ParticleFlow/interface/egamma/pftkegsorter_ref.h

```diff
@@ -5,12 +5,10 @@
 #include <vector>
 
 #include "DataFormats/L1TParticleFlow/interface/layer1_emulator.h"
+#include "L1Trigger/Phase2L1ParticleFlow/interface/dbgPrintf.h"
 
 #ifdef CMSSW_GIT_HASH
```

Comment:
Just my curiosity, why do we need `CMSSW_GIT_HASH` to control ParameterSet header file?

Reply:
@gpetruc Can you address this question?